simcase-game/src/components/Header.js

```diff
@@ -3,11 +3,10 @@ import React from "react";
 import { jsx } from "@emotion/core";
 import { createPortal } from "react-dom";
 import { Dialog } from "@reach/dialog";
-import "@reach/dialog/styles.css";
-import Role from "../views/Role";
-import Leaderboard from "../views/Leaderboard";
-import Walkthrough from "../views/Walkthrough";
 import { navigate } from "@reach/router";
+import MyModal from "../views/Modal";
```

Comment:
What do you think about giving a more descriptive name like ModalRanking or something like so?
I would prefer have the name of the component you are exporting equal to the name of the function that creates the component.

Reply:
Done, changed the name from Modal to ReusableModal which is a better name.